src/app/modules/game/pages/lobby/lobby.component.scss

```diff
@@ -1,9 +1,117 @@
+@import "plugins/variables";
+@import "plugins/mixins";
+
 .lobby {
+  width: 100%;
+  &__container {
+    background-image: url(/assets/backgrounds/lobby-background.gif);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+    top: 0;
+    margin: 0;
+    padding: 0;
+    background-size: 90% 100%;
+    height: 100vh;
+    position: relative;
+  }
+  &__back {
+    padding: 0.2rem 1rem;
```

Comment:
magic numbers here. em and rem its not a best units of measurement, i think better use % or px instead

Reply:
Agreed with @maxonchil to leave as is.